### ISSUE TITLE
MXIII 2/8 boot loader locked

### ISSUE THREAD

**Issue by Kezzi** [codesnake/OpenELEC.tv#82]
anyone know how to flash openelec on this box with android 4.4.2
They have locked out the boot loaders in all their device builds?
Help please I. Would love to get this box re Flashed! Probably many others as well!
Thank you!

**Comment by fratam** [comment]
Mail me @ fratam@chello.nl please and i Will help you

**Comment by Kezzi** [comment]
Ok Great I can use any help!!!Thank you for trying!!!

**Comment by fratam** [comment]
Do you have whatsapp?
This is my nummer
0648135743
Op 5 sep. 2015 16:20 schreef "Jon" <notifications@github.com>:

**Comment by Kezzi** [comment]
I have an IPad what is what's App?

Sent from my iPad

On Sep 5, 2015, at 10:27 AM, fratam <notifications@github.com<mailto:notifications@github.com>> wrote:

Do you have whatsapp?
This is my nummer
0648135743
Op 5 sep. 2015 16:20 schreef "Jon" <notifications@github.com<mailto:notifications@github.com>>:

> Ok Great I can use any help!!!Thank you for trying!!!
>
> -
> Reply to this email directly or view it on GitHub
> <https://github.com/codesnake/OpenELEC.tv/issues/82#issuecomment-137959931>
> .
>

-

**Comment by Kezzi** [comment]
Thank you Fratman!!!Solution for New MX3 with Locked Bootloaders.
Extract the desired compilation to a Fat32 formatted SDcard...Plug in the Power to the MX...now you must hold the Toothpick In and at the same time Press and hold the ON/OFF Button...after about 5 to 6 Seconds Push in the SDcard....wait for the Green Man to show and release both Toothpick and On/Off.
You may get a black screen after....Restart with Power wire if the system is still Black Do it Again!!!
This works very well!!! I hope it helps!!!

**Comment by maso76** [comment]
Hi Kezzi - I tried this method with locked bootloaders.   got the green man, then screen went blank at the end of progress bar, but I can't get the system re-started.  Red power light when I power off / on again but won't turn on / turn to blue power light.

any suggestions?
thanks

**Comment by fratam** [comment]
Yes first flash your box back to firmware 1114k with THE USB tool en USB
otg cable !


Op do 17 sep. 2015 om 15:54 schreef maso76 <notifications@github.com>

**Comment by fratam** [comment]
http://www.mediafire.com/download/5g9w482z44bouhe/USB_Burning_Tool_v2.0.4_x86.rar
Op do 17 sep. 2015 om 18:55 schreef mst radio <frankmstradio@gmail.com>

**Comment by fratam** [comment]
THEn try again THE first method
Op do 17 sep. 2015 om 18:56 schreef mst radio <frankmstradio@gmail.com>

**Comment by fratam** [comment]
I always first flash te boxes Back to this firmware and THEn put THE new
OpenELEC on THE boxes
Op do 17 sep. 2015 om 18:57 schreef mst radio <frankmstradio@gmail.com>

**Comment by maso76** [comment]
thanks!  I'm not 100% certain of the exact model.  My box has Amlogic S805 chipset and says "mxq tonbux" on it and has the AV port next to the HDMI port.
http://www.amazon.co.uk/gp/aw/d/B00WWIUIN2/ref=mw_dp_img_4?in=4&is=l

any thoughts on / links to best original firmware to try with it please?
and best openelec to flash?

many thanks

**Comment by fratam** [comment]
Take THE tv110 from here

http://amlinux.ru/image/5.0.6/
Op do 17 sep. 2015 om 23:17 schreef maso76 <notifications@github.com>

**Comment by fratam** [comment]
This is an older forum file
And there they say s805 is tv110

Op vr 18 sep. 2015 om 00:37 schreef mst radio <frankmstradio@gmail.com>

**Comment by Kezzi** [comment]
Yes  Try it again it may take a few tries! Timing the insertion of the sad card is a little tricky but it works!

Sent from my iPad

On Sep 17, 2015, at 9:54 AM, maso76 <notifications@github.com<mailto:notifications@github.com>> wrote:


Hi Kezzi - I tried this method with locked bootloaders. got the green man, then screen went blank at the end of progress bar, but I can't get the system re-started. Red power light when I power off / on again but won't turn on / turn to blue power light.

any suggestions?
thanks

-

**Comment by Kezzi** [comment]
You must find the Openelec that has both the 802 and the 805 in its' title !

Sent from my iPad

On Sep 17, 2015, at 5:17 PM, maso76 <notifications@github.com<mailto:notifications@github.com>> wrote:


thanks! I'm not 100% certain of the exact model. My box has Amlogic S805 chipset and says "mxq tonbux" on it and has the AV port next to the HDMI port.
http://www.amazon.co.uk/gp/aw/d/B00WWIUIN2/ref=mw_dp_img_4?in=4&is=l

any thoughts on / links to best original firmware to try with it please?
and best openelec to flash?

many thanks

-

**Comment by Kezzi** [comment]
I turned on the security to allow other software etc and turned off the check!
In settings go to the version and click on version number 7 times to activate Developer settings and then click on Return.. Now go to developer options and turn on USB and SD use and turn off any security check... Now try the install and it will Work!

Sent from my iPad

On Sep 17, 2015, at 12:57 PM, fratam <notifications@github.com<mailto:notifications@github.com>> wrote:

THEn try again THE first method
Op do 17 sep. 2015 om 18:56 schreef mst radio <frankmstradio@gmail.com<mailto:frankmstradio@gmail.com>>

>
> http://www.mediafire.com/download/5g9w482z44bouhe/USB_Burning_Tool_v2.0.4_x86.rar
> Op do 17 sep. 2015 om 18:55 schreef mst radio <frankmstradio@gmail.com<mailto:frankmstradio@gmail.com>>
>
>> Yes first flash your box back to firmware 1114k with THE USB tool en USB
>> otg cable !
>>
>>
>> Op do 17 sep. 2015 om 15:54 schreef maso76 <notifications@github.com<mailto:notifications@github.com>>
>>
>>> Hi Kezzi - I tried this method with locked bootloaders. got the green
>>> man, then screen went blank at the end of progress bar, but I can't get the
>>> system re-started. Red power light when I power off / on again but won't
>>> turn on / turn to blue power light.
>>>
>>> any suggestions?
>>> thanks
>>>
>>> -
>>> Reply to this email directly or view it on GitHub
>>> <https://github.com/codesnake/OpenELEC.tv/issues/82#issuecomment-141093868>
>>> .
>>>
>>

-
